torch/ao/quantization/quantize_fx.py

```diff
@@ -99,6 +111,8 @@ def _prepare_fx(
             "in a future version. Please pass in a PrepareCustomConfig instead.")
         prepare_custom_config = PrepareCustomConfig.from_dict(prepare_custom_config)
 
+    _attach_meta_to_node_if_not_exist(model)
```

Comment:
do you need this here?

Reply:
oh this should be after tracing, will move

Reply:
I think since you call in `_fuse_fx` you don't need to call it twice.

Reply:
oh, I should put that in `_fuse_fx` I think, will move that one as well